mbuild/compound.py

```diff
@@ -82,8 +80,13 @@ def load(filename_or_object, relative_to_module=None, compound=None, coords_only
     type_dict = {
         pmd.Structure:compound.from_parmed,
         md.Trajectory:compound.from_trajectory,
-        pybel.Molecule:compound.from_pybel,
     }
+    try:
```

Comment:
Looking through the discussion, it makes sense for the change in import style, maybe we want to have a non-error generating `import_` method. Most of the times these should error out, but in cases like this it might be useful?

Curious what others think?

Reply:
Worthy of a discussion, maybe there should be a less strict sister function. Could you please raise an issue on this?